util/importers/LDC97S62.py

```diff
@@ -102,50 +87,70 @@ def next_batch(self):
 
     @property
     def total_batches(self):
-        # Note: If len(_txt_files) % batch_size != 0, this re-uses initial _txt_files
-        return int(ceil(float(len(self._txt_files)) / float(self.batch_size)))
-
-
-def read_data_sets(data_dir, train_batch_size, dev_batch_size, test_batch_size, numcep, numcontext, thread_count=8, stride=1, offset=0, next_index=lambda s, i: i + 1, limit_dev=0, limit_test=0, limit_train=0, sets=[]):
-    data_dir = os.path.join(data_dir, "LDC97S62")
-
-    # Conditionally convert swb sph data to wav
-    _maybe_convert_wav(data_dir, "swb1_d1", "swb1_d1-wav")
-    _maybe_convert_wav(data_dir, "swb1_d2", "swb1_d2-wav")
-    _maybe_convert_wav(data_dir, "swb1_d3", "swb1_d3-wav")
-    _maybe_convert_wav(data_dir, "swb1_d4", "swb1_d4-wav")
-
-    # Conditionally split wav data
-    _maybe_split_wav(data_dir, "swb_ms98_transcriptions", "swb1_d1-wav",
-                     "swb1_d1-split-wav")
-    _maybe_split_wav(data_dir, "swb_ms98_transcriptions", "swb1_d2-wav",
-                     "swb1_d2-split-wav")
-    _maybe_split_wav(data_dir, "swb_ms98_transcriptions", "swb1_d3-wav",
-                     "swb1_d3-split-wav")
-    _maybe_split_wav(data_dir, "swb_ms98_transcriptions", "swb1_d4-wav",
-                     "swb1_d4-split-wav")
-
-    _maybe_split_transcriptions(data_dir, "swb_ms98_transcriptions")
-
-    _maybe_split_sets(data_dir, ["swb1_d1-split-wav", "swb1_d2-split-wav", "swb1_d3-split-wav", "swb1_d4-split-wav"],
-                      "final_sets")
+        # Note: If len(_filelist) % batch_size != 0, this re-uses initial files
+        return int(ceil(len(self._filelist) / self.batch_size))
```

Comment:
The import
```python
from __future__ import absolute_import, division, print_function
```
changes the value returned here.

For example if `len(self._filelist)` were 10 and `self.batch_size` were 6, then in python 2.x one finds
```bash
>>> from math import ceil
>>> int(ceil(10 / 6))
1
>>> from __future__ import division
>>> int(ceil(10 / 6))
2
```
So this changes the `total_batches` property.